gnomad/resources/grch38/gnomad.py

```diff
@@ -517,13 +521,20 @@ def gnomad_gks(
 
     # Select relevant fields, checkpoint, and filter to interval before adding
     # annotations
-    ht = ht.select(ht.freq, ht.info.vrs, ht.popmax)
-    ht = ht.checkpoint(hl.utils.new_temp_file("vrs_checkpoint", extension="ht"))
+    if not skip_coverage:
+        ht = ht.select(ht.freq, ht.info.vrs, ht.popmax, ht.mean_cov)
+    else:
+        ht = ht.select(ht.freq, ht.info.vrs, ht.popmax)
+
+    # Checkpoint narrower set of columns if not skipped.
+    if not skip_checkpoint:
+        ht = ht.checkpoint(hl.utils.new_temp_file("vrs_checkpoint", extension="ht"))
     ht = hl.filter_intervals(ht, [locus_interval])
 
     # Collect all variants as structs, so all dictionary construction can be
     # done in native Python
     variant_list = ht.collect()
+    ht_freq_index_dict = ht.freq_index_dict.collect()[0]
```

Comment:
remind me, what was the data you had said for how much this speeds things up? I'm not clear how long this collect statement was taking for each variant

Reply:
nevermind I see your comments above